.changeset/ten-plums-flow.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
will this be empty?

Reply:
Yes, I just added it to pass the CI cases. I can say it's a patch change for app to add documentation if needed.

Reply:
@Zaperex Please only add one changeset.

Reply:
Ah my mistake, I thought I was editting the old one

Reply:
Fixed